content/public/android/java/src/org/chromium/content/browser/selection/SelectionPopupControllerImpl.java

```diff
@@ -250,6 +250,61 @@ public interface ReadbackViewCallback {
         View getReadbackView();
     }
 
+    private Delegate delegate_;
```

Comment:
should be called mDelegate to be consistent with the rest of java code.

Reply:
Ah, right. Done!